tests/setup_test_path.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```

Comment:
Is this supposed to be imported somewhere? Unless it's guaranteed imported first it doesn't seem like it will ensure anything about import order?